docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 * :feature:`2507` Users can now delete imported trades and deposit/withdrawals from crypto.com via the purge data UI.
 * :bug:`2530` Poloniex should no longer display phantom LEND balances in rotki.
 * :bug:`2534` Aave v2 tokens not in Aave v1 should no longer have their balance double counted.
+* :bug:`2539` Now the effects of add/edit/remove operations related to ledger actions are committed to the database.
```

Comment:
This entry is correct as far as developing is concerned. But for our changelog as you can see from other entries we try to follow a more user-centric wording. So here I would suggest the following:

```suggestion
* :bug:`2539` The effects of adding/editing/removing a ledger actions will no longer be lost if rotki restarts right after the operation.
```

The idea is the user does not understand anything about DBs and stuff. But understands the bug being, to add something and then restart rotki, and then not see the added action

Reply:
I see. Will take this into account for future modifications on the changelog